Tem-Test/server.R

```diff
@@ -5,22 +5,21 @@ library(ggplot2)
 server<- function(input,output) {
   dataset<- reactive({
     selected_data<- data_list[as.integer(input$dataset),]
-    browser()
-    get(selected_data["Item"],pasteO("package:", selected_data["Package"]))
```

Comment:
This took a while to find-- turns out you had `pasteO()` instead of `paste0()`. Typo.